fesvr/htif.cc

```diff
@@ -35,7 +35,6 @@
 # define TARGET_DIR "/" TARGET_ARCH "/bin/"
 #endif
 
-static volatile bool signal_exit = false;
```

Comment:
What do the changes in this file have to do with adding APIs? If this is legit it needs to be a separate commit with a message explaining...